tests/test_api.py

```diff
@@ -194,13 +196,25 @@ def test_reporter_with_branches(self):
         sh.coverage('run', '--branch', 'runtests.py')
         results = self.cover.get_coverage()
         assert len(results) == 2
+
+        branches = results[0]['branches']
+        assert not len(branches) % 4
```

Comment:
Took me a second to figure out why this is (why branches must be divisible by four). Could you please add a comment here?